.github/ISSUE_TEMPLATE/crash-issues.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Crash Report
+about: For issues regarding game crashes or permanent freezes
+---
+
+<!-- After you fill in all information, delete all comments in the issue -->
+
+**Describe your problem:** <!-- Provide any information you believe could be useful -->
+
+**Screenshots or videos showing encountered issue:** 
+
+**osu!lazer version:** <!-- Provide the version of your osu!lazer, you can find it at the bottom of the screen  -->
+
+**Logs:** <!-- Attach your osu!lazer logs, you can find them under %appdata%\osu\logs in Windows, or under ~/.local/share/osu/ in Linux and macOS -->
+
+**Computer Specifications:** <!-- Attach your computer specifications, you can find them by using Speccy -->
```

Comment:
~`Speccy`~
Should be something that is already integrated in the OS, not insisting people to download that.

Reply:
@Joehuu I doubt you can find any specs checkers built  in for other OSes like Linux however.

but for Windows, System Details should suffice.